src/app/routes/topic/getInitialData/index.test.js

```diff
@@ -1,8 +1,8 @@
 import getInitialData from '.';
 
 describe('get initial data for topic', () => {
-  it('should return our title', () => {
+  it.skip('should return our title', () => {
```

Comment:
Any reason this test is skipped at this stage?

Reply:
Yes, we are not dealing with the data in this PR. We are continuing to use the fixture data. The ticket for using the BFF data is here - https://github.com/bbc/simorgh-infrastructure/issues/1543